java/spacewalk-java.changes

```diff
@@ -1,3 +1,5 @@
+- Manage reboot in transactional update action chain (bsc#1201476)
+- Upgrade Bootstrap to 3.4.1
```

Comment:
seems a wrong changelog entry sneaked in

Reply:
fixed